reana_client/cli/workflow.py

```diff
@@ -903,9 +903,9 @@ def workflow_open_interactive_session(ctx, workflow, access_token, image,
                 "Opening an interactive session on {}".format(workflow))
             path = open_interactive_session(workflow, access_token, image,
                                             port)
-            click.secho("{reana_server_url}{path}".format(
+            click.secho("{reana_server_url}{path}?token={access_token}".format(
                 reana_server_url=ctx.obj.reana_server_url,
-                path=path), fg="green")
+                path=path, access_token=access_token), fg="green")
```

Comment:
After ` $ reana-client open workflow.1` returned an url,  it took ~ 4 mins to start container. What do you think about adding info message `INFO: It could take several minutes to start interactive session.`?